docs/docker-dev/docker-dev-setup-manual.md

```diff
@@ -0,0 +1,18 @@
+# Docker Development Environment Setup
+
+1. Install [Docker](https://docs.docker.com/get-docker/) if not already.
+2. On the terminal, run command `curl -o- https://raw.githubusercontent.com/opensearch-project/OpenSearch-Dashboards/main/dev-tools/install-docker-dev.sh | bash`. This should create a folder named `opensearch-dashboards-docker-dev` and it should contain two files: `docker-compose.yml` and `entrypoint.sh`.
```

Comment:
Other steps use "in the terminal". We should be consistent, and "in" is likely more commonly used

```suggestion
2. On the terminal, run the command `curl -o- https://raw.githubusercontent.com/opensearch-project/OpenSearch-Dashboards/main/dev-tools/install-docker-dev.sh | bash`. This should create a folder named `opensearch-dashboards-docker-dev` and it should contain two files: `docker-compose.yml` and `entrypoint.sh`.
```

We should also provide a plain link to the installer script for people to quick inspect.

Reply:
Changed the syntax, also added the plain link to the installer script.